app/src/androidTest/java/com/google/jetpackcamera/ImageCaptureDeviceTest.kt

```diff
@@ -152,6 +154,147 @@ internal class ImageCaptureDeviceTest {
         Truth.assertThat(doesImageFileExist(uri, "video")).isFalse()
     }
 
+    @Test
+    fun multiple_image_capture_external() {
+        val timeStamp = System.currentTimeMillis()
+        val uriStrings = arrayListOf<String>()
+        for (i in 1..3) {
+            val uri = getTestUri(DIR_PATH, timeStamp + i.toLong(), "jpg")
+            uriStrings.add(uri.toString())
+        }
+        val result =
+            runScenarioTestForResult<MainActivity>(
+                getMultipleImageCaptureIntent(
+                    uriStrings,
+                    MediaStore.INTENT_ACTION_STILL_IMAGE_CAMERA
+                )
+            ) {
+                // Wait for the capture button to be displayed
+                composeTestRule.waitUntil(timeoutMillis = APP_START_TIMEOUT_MILLIS) {
+                    composeTestRule.onNodeWithTag(CAPTURE_BUTTON).isDisplayed()
+                }
+
+                composeTestRule.onNodeWithTag(CAPTURE_BUTTON)
+                    .assertExists()
+                    .performClick()
+                composeTestRule.waitUntil(timeoutMillis = IMAGE_CAPTURE_TIMEOUT_MILLIS) {
+                    composeTestRule.onNodeWithTag(IMAGE_CAPTURE_SUCCESS_TAG).isDisplayed()
+                }
+                composeTestRule.waitUntil(timeoutMillis = APP_START_TIMEOUT_MILLIS) {
```

Comment:
You should create a different timeout constant and not use `APP_START_TIMEOUT_MILLIS` for waiting for the toast to go away.